src/meson.build

```
@@ -510,7 +514,17 @@ endif
 if conf.get('HAVE_CORETEXT', 0) == 1
   hb_sources += hb_coretext_sources
   hb_headers += hb_coretext_headers
-  harfbuzz_deps += coretext_deps
+  harfbuzz_deps += coretext_dep
```

Looks like a typo was introduced here.
```suggestion
  harfbuzz_deps += coretext_deps
```